wayback/_client.py

```diff
@@ -712,6 +717,7 @@ def search(self, url, *, match_type=None, limit=1000, offset=None,
             sent_query, next_query = next_query, None
             with _utils.rate_limited(self.session.search_calls_per_second,
                                      group='search'):
+                logger.debug("querying %s %s", CDX_SEARCH_URL, sent_query)
```

Comment:
Hmmm, do you think this is useful when the first thing `WaybackSession.send()` does is log the request?

Reply:
Good point, I'll remove it.